lib/core/elastic_connector_actions.rb

```diff
@@ -157,28 +157,25 @@ def claim_job(connector_id)
           seq_no,
           primary_term
         )
+      end
 
+      def create_job(connector_settings:)
         body = {
-          :status => Connectors::SyncStatus::IN_PROGRESS,
-          :worker_hostname => Socket.gethostname,
-          :created_at => Time.now,
-          :started_at => Time.now,
-          :last_seen => Time.now,
-          :connector => {
-            :id => connector_id,
-            :filtering => convert_connector_filtering_to_job_filtering(connector_record.dig('_source', 'filtering'))
+          status: Connectors::SyncStatus::PENDING,
+          worker_hostname: Socket.gethostname,
```

Comment:
`worker_hostname` should only be added when it's claimed. It's the machine which executes the job, not the one which enqueues the job.

Reply:
Good point